Description: program_options regression in 1.49.0?
 Since 1.49, bitcoin unable to compile with error [...]
 Steps to avoid error:

    Error can be avoided by removing #if around #include <.../convert.hpp> in /usr/include/boost/program_options/detail/config_file.hpp 
Author: Konstantin Nikiforov 
Bug: 671409
Forwarded: https://svn.boost.org/trac/boost/ticket/6790

--- a/boost/program_options/detail/config_file.hpp	2012-03-30 01:14:58.000000000 -0500
+++ b/boost/program_options/detail/config_file.hpp	2012-05-04 19:13:53.120697282 -0500
@@ -17,9 +17,7 @@
 #include <boost/program_options/eof_iterator.hpp>
 
 #include <boost/detail/workaround.hpp>
-#if BOOST_WORKAROUND(__MWERKS__, BOOST_TESTED_AT(0x3202))
 #include <boost/program_options/detail/convert.hpp>
-#endif
 
 #if BOOST_WORKAROUND(__DECCXX_VER, BOOST_TESTED_AT(60590042))
 #include <istream> // std::getline
